kong/plugins/jwt-claims-to-headers/schema.lua

```diff
@@ -8,34 +9,55 @@ defaultHeaderPrefix = "X-Jwt-Claim-"
 -- be used in this plugin, to tell the plugin where to find the jwt.
 
 return {
-    no_consumer = true, -- this plugin will only be applied to Services or Routes,
+    name = "JWT claims to headers",
     fields = {
-        header_prefix = {
-            type = "string",
-            default = defaultHeaderPrefix
+        {
+          -- this plugin will only be applied to Services or Routes
+          consumer = typedefs.no_consumer
```

Comment:
are you sure consumer should be a field attribute? It isn't in the code right now!

Reply:
We had `no_consumer = true,` and looks like this is the new definition

Reply:
looks good. Just confirmed from the documentation!!